tools/run_tests/dockerize/build_interop_image.sh

```diff
@@ -75,6 +75,22 @@ git branch -f jenkins-docker
 
 CONTAINER_NAME="build_${BASE_NAME}_$(uuidgen)"
 
+# build_interop.sh could be missing from old git tree and we will have issue unless we find the proper
+# build_interop.sh to use in such case.
+if ! [ -e $GRPC_ROOT/tools/dockerfile/interoptest/$BASE_NAME/build_interop.sh ]; then
```

Comment:
I am confused by this snippet. 
- it seems completely out of context in this file (and the comment won't really help to understand that this was added to the interop tests to enable the interop matrix flow
- it looks like you expect to have grpc_interop_cxx_debian8/ directory with Dockerfile, but the build_interop.sh will be missing? How would that happen?